src/atlas/domain/sequence_diagram.clj

```diff
@@ -72,6 +72,24 @@
              :label      "response"})
       acc)))
 
+(defn- producer-span->arrow [trace]
+  (fn [acc producer-span]
+    (conj acc
+          {:id (:span-id producer-span)
+           :from (span->service-name producer-span trace)
+           :to (span->topic producer-span)
+           :start-time (microseconds->epoch (:start-time producer-span))
+           :label "produce"})))
+
+(defn- consumer-span->arrow [trace]
+  (fn [acc consumer-span]
+    (conj acc
+          {:id (:span-id consumer-span)
+           :from (span->topic consumer-span)
+           :to (span->service-name consumer-span trace)
+           :start-time (microseconds->epoch (:start-time consumer-span))
```

Comment:
honestly, I'm not sure about this. I was wondering if we should use `span->end-time` maybe? What do you think?

Reply:
hmmm also not sure. I'd rather let this way and revisit in the future if we need to